src/index.js

```diff
@@ -1,4 +1,4 @@
 import UserAgent from './user-agent';
 
 
-module.exports = UserAgent;
+export default UserAgent;
```

Comment:
I think there is a better solution, because this change break the require statement already in use.

Reply:
@sangaline do you have an idea for addressing this one ? This why I doesn't use babel with nodejs

Reply:
After digging it's not possible https://stackoverflow.com/questions/40294870/module-exports-vs-export-default-in-node-js-and-es6#40295288

So I suggest we keep the old behavior. And export as CommonJs not ES6 Module.

Reply:
In #36, I integrated [babel-plugin-add-module-exports](https://www.npmjs.com/package/babel-plugin-add-module-exports) which seemed to work with both `import` and `require()` syntax in my tests. Are you running into any breakage or change in behavior following that update?